src/pages/booklist/containers/BooklistContainer.tsx

```diff
@@ -51,18 +48,16 @@ export function BooklistContainer(): JSX.Element | null {
               .map(({ sections, subject, code }) => {
                 return <TextbookCard subject={subject} code={code} sections={sections} />;
               })
+          ) : textbooks.textbookInfo.filter((textbook) => textbook && textbook.term === term).length <= 0 &&
+            textbooks.textbookInfo.length <= 0 ? (
+            <NotFound item="Unable to find saved courses from your" term={term} timetableButton timetable />
```

Comment:
Is there a case where `timetable != timetableButton`? I think these can probably be merged to one prop `timetable`

Reply:
Good point, just made the change 👍. I don't think there will be a case in the future when they don't go together either.